lib/crypto.test.ts

```
@@ -0,0 +1,497 @@
+          clearEncryptionKeyCache();
+        }
+      });
+
+      it('should handle key changing to same value (no re-derivation)', () => {
+        const text = 'same-value-test';
+        const encrypted1 = encryptToken(text);
+
+        // eslint-disable-next-line no-self-assign
+        process.env.OAUTH_ENCRYPTION_KEY = process.env.OAUTH_ENCRYPTION_KEY;
+        // eslint-disable-next-line no-self-assign
+        process.env.OAUTH_ENCRYPTION_SALT = process.env.OAUTH_ENCRYPTION_SALT;
```

Biome (2.1.2)

[error] OAUTH_ENCRYPTION_KEY is assigned to itself.

This is where is assigned.

Self assignments have no effect and can be removed.

(lint/correctness/noSelfAssign)